Realm/RLMMigration.h

```diff
@@ -25,91 +25,97 @@ NS_ASSUME_NONNULL_BEGIN
 @class RLMObject;
 
 /**
-Provides both the old and new versions of an object in this Realm. Object properties can only be
-accessed using keyed subscripting.
-
-@param oldObject Object in original RLMRealm (read-only).
-@param newObject Object in migrated RLMRealm (read-write).
+ A block type which provides both the old and new versions of an object in the Realm. Object 
+ properties can only be accessed using keyed subscripting.
+ 
+ @see `-[RLMMigration enumerateObjects:block:]`
+ 
+ @param oldObject The object from the original Realm (read-only).
+ @param newObject The object from the migrated Realm (read-write).
 */
 typedef void (^RLMObjectMigrationBlock)(RLMObject * __nullable oldObject, RLMObject * __nullable newObject);
 
 /**
- RLMMigration is the object passed into a user defined RLMMigrationBlock when updating the version
- of an RLMRealm instance.
-
- This object provides access to the RLMSchema current to this migration.
+ `RLMMigration` instances encapsulate information intended to facilitate a schema migration.
+ 
+ A `RLMMigration` instance is passed into a user-defined `RLMMigrationBlock` block when updating
+ the version of a Realm. This instance provides access to the old and new database schemas.
```

Comment:
It does a lot more than that. Something like "This instance provides access to the old and new database schemas, the objects in the Realm, and provides functionality for modifying the Realm during the migration".